registration/models.py

```diff
@@ -371,6 +371,7 @@ def commit_changes_to_instance(self, deploy_on_commit=False, retry_attempts=2, c
         instance.name = self.instance_name
         instance.privacy_policy_url = self.privacy_policy_url
         instance.email = self.public_contact_email
+        instance.creator_id = creator
```

Comment:
This is not the best place to do this since it will overwrite the instance creator every time a new deployment is triggered from the new UI. While I think right now only the instance creator can trigger deployments from the new UI, that might not be the case in the future.

The appropriate place to set the creator would be when the instance is first created, which happens inside [this method](https://github.com/open-craft/opencraft/blob/ebd8c3666388173174f3fdf7c289a39104f61f90/registration/provision.py#L55).

Reply:
Also when we move this code to the more appropriate place, can we add a unit test to verify it works?

Reply:
Done! I didn't know the flow of execution but you are, it doesn't make sense to add it on every new deployment. I also add the test you asked for